docs/en/observability/monitor-nginx.asciidoc

```diff
@@ -9,7 +9,7 @@
 Return to this tutorial after you've learned the basics.
 ****
 
-Use the https://docs.elastic.co/integrations/nginx[Nginx Elastic integration] and the {agent} to collect valuable metrics and logs from your nginx instances. Then, use built-in dashboards and tools like Logs Explorer in {kib} allow you to visualize and monitor your nginx data from one place. This data provides valuable insight into your nginx instances—for example:
+Use the {integrations-docs}/nginx[Nginx Elastic integration] and the {agent} to collect valuable metrics and logs from your nginx instances. Then, use built-in dashboards and tools like Logs Explorer in {kib} allow you to visualize and monitor your nginx data from one place. This data provides valuable insight into your nginx instances—for example:
```

Comment:
It's a trivial point, but I'm not understanding the rule you are following to capitalize nginx.

Reply:
I was trying to use the lowercase (one of nginx's trademarked versions of their name) except in specific UI instances, but I missed a couple it looks like. Should be updated now.